### ISSUE TITLE
[Feature Request] Custom conversion operator symbol

### ISSUE THREAD

**Issue by ggdorman** [MM2-0/Kvaesitso#136]
It would be great if we could enter our own custom symbol to use as the conversion symbol, as typing ">>" on a phone keyboard is typically slow to do, especially repeatedly.

**Comment by MM2-0** [comment]
True. I was looking for a symbol that would work in any language but ">>" it's probably not the best solution. What would you choose instead?

**Comment by ggdorman** [comment]
That's a good question. ">>" is definitely logical and works in any language. To be honest I'm not sure any of the symbols on the second page of (my) keyboard would make too much sense. Maybe the "@" key or a colon would work. An alternative solution could be to add a button next to/below the search bar to insert the conversion symbol. Thoughts?

**Comment by MM2-0** [comment]
I prefer to not add any more buttons to the search bar, so unless there is an API to add this button to the keyboard (which I don't think there is) this is not an option for me.

In most cases it's probably fastest anyway to leave out the target unit and find the desired conversion in the list.

**Comment by jack3308** [comment]
You could set it as a user defined field in the settings maybe?? Idk of that'd work or not - maybe just tell people thagbif they have issues with what they chose then thats on them and to choose something different?
